src/extension/content-script/liquid.js

```diff
@@ -76,6 +77,10 @@ async function init() {
           }
         }
 
+        if (ev.data.action === `${SCOPE}/isEnabled`) {
```

Comment:
Now we use `${SCOPE}` it would be nice to reduce duplication in these content script provider files as a followup PR. What do you think?

Reply:
yup in some files there is scope variable declared and in some there is not. i will clean it up a bit in follow up pr

Reply:
these are very critical, a mistake must not be made. thus I kinda actually also like hardcoded stuff here.